javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var $lightbulb = document.querySelector('.column');
+var $container = document.querySelector('.container');
+
+$lightbulb.addEventListener('click', toggleLightbulb);
+
+var onOff = 2;
+
+function toggleLightbulb(event) {
+  onOff++;
+  if (onOff % 2 === 1) {
+    $lightbulb.className = 'column ' + 'off';
+    $container.className = 'container ' + 'off-container';
+  } else {
+    $lightbulb.className = 'column ' + 'on';
```

Comment:
No need for concatenation here, you can just have the values all together in one string